ui/app/registration/views/search.html

```diff
@@ -26,8 +26,8 @@
                     <input tabindex="5" type="text" id="customAttribute" ng-model="searchParameters.customAttribute" placeholder="{{::customAttributesSearchConfig.placeholder}}">
                 </article>
                 <article class="fl" ng-if="::addressSearchConfig.show">
-                    <label for="name"><strong>{{::addressSearchConfig.label}}</strong></label>
-                    <input tabindex="6" type="text" id="addressFieldValue" ng-model="searchParameters.addressFieldValue" placeholder="{{::addressSearchConfig.placeholder}}">
+                    <label for="name"><strong>{{::'REGISTRATION_SEARCH_ADDRESS_LABEL_KEY' | translate}}</strong></label>
```

Comment:
@rubailly the keys for the address search label and placeholder are configured in `registration/app.json` (under "patientSearch"). These key's values are available in the scope variables addressSearchConfig.label and addressSearchConfig.placeholder. We can therefore directly translate the scope variables (I.e. `{{::addressSearchConfig.label | translate }}`); this way we do not need to hard code the keys in the code (with the current solution, if the key changes in app.json, the translation will break). 

@ajinkyagadewar  this fix could benefit Bahmni as well, can we arrange to merge this back into Bahmni?

Reply:
@tmvumbi2 - this has been completed